data-es2016plus.js

```diff
@@ -1982,11 +1982,17 @@ exports.tests = [
     ]
   },
   {
-    name: 'Proxy "ownKeys" handler, duplicate keys for non-extensible targets (ES Draft style)',
+    name: 'Proxy "ownKeys" handler, duplicate keys for non-extensible targets (ES 2017 semantics)',
```

Comment:
I believe "Old ES2017 semantics" would be clearer.

Reply:
This change (https://github.com/tc39/ecma262/issues/833) was incorporated into ES2017. Were the prior semantics introduced only after ES2016?

Reply:
Ahh right thanks for clarifying.

The. “ES2017 semantics” and “ES2018 semantics” seem like good names?

Reply:
"ES2017 semantics" sounds new to me, which can be misleading.  How about "Deprecated ES2017 semantics"?

Reply:
If there’s a 2018 test next to it, 2017 wouldn’t sound replaced?

Reply:
I thought the ES2017 test would be moved to non-standard, and the ES2018 would be implemented here...

Reply:
Technically the ES2017 semantics *are* standard; they've just been superseded by a later one. I can see the argument for putting them in either place.

Reply:
If they are both on the same tab, it will be impossible to reach 100% on the tab.

Reply:
Fair point; that's an argument to move it to the non-standard tab. However, I don't particularly want to be optimizing for "reaching 100%" :-/

Reply:
True, but I don't want to create a test that claims "passing" if an engine never implements the 2018 semantics.